database/src/migrations/20210611115350_block_observation.ts

```diff
@@ -0,0 +1,95 @@
+import Knex from 'knex';
+
+const DB_SCHEMA = process.env.DB_SCHEMA;
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`
+
+  GRANT ALL ON SCHEMA biohub TO postgres;
+
+  set schema '${DB_SCHEMA}';
+  set search_path = ${DB_SCHEMA},public;
+
+--
+-- TABLE: block_observation
+--
+
+CREATE TABLE ${DB_SCHEMA}.block_observation(
+    id                integer           GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    b_id              integer           NOT NULL,
+    s_id              integer           NOT NULL,
+    start_time        timestamptz(6)    DEFAULT now() NOT NULL,
+    end_time          timestamptz(6)    DEFAULT now() NOT NULL,
+    observation_cnt   integer           NOT NULL,
+    data              json              NOT NULL,
+    create_date       timestamptz(6)    DEFAULT now() NOT NULL,
+    create_user       integer           NOT NULL,
+    update_date       timestamptz(6),
+    update_user       integer,
+    revision_count    integer           DEFAULT 0 NOT NULL,
+    CONSTRAINT "PK170" PRIMARY KEY (id)
+)
+;
+
+
+    COMMENT ON COLUMN block_observation.id IS 'System generated surrogate primary key identifier.'
+    ;
+    COMMENT ON COLUMN block_observation.b_id IS 'System generated surrogate primary key identifier.'
+    ;
+    COMMENT ON COLUMN block_observation.start_time IS 'The datetime the observation was started.'
+    ;
+    COMMENT ON COLUMN block_observation.end_time IS 'The datetime the observation ended.'
+    ;
+    COMMENT ON COLUMN block_observation.observation_cnt IS 'The computed total of the observations.'
+    ;
+    COMMENT ON COLUMN block_observation.data IS 'The json data associated with the record.'
+    ;
+    COMMENT ON COLUMN block_observation.create_date IS 'The datetime the record was created.'
+    ;
+    COMMENT ON COLUMN block_observation.create_user IS 'The id of the user who created the record as identified in the system user table.'
+    ;
+    COMMENT ON COLUMN block_observation.update_date IS 'The datetime the record was updated.'
+    ;
+    COMMENT ON COLUMN block_observation.update_user IS 'The id of the user who updated the record as identified in the system user table.'
+    ;
+    COMMENT ON COLUMN block_observation.revision_count IS 'Revision count used for concurrency control.'
+    ;
+    COMMENT ON TABLE block_observation IS 'A persistent store for draft block_observation data. For example, if a user starts a project creation process and wants to save that information as a draft then the block observation data can be persisted for subsequent reload into the project creation process.'
```

Comment:
I dont think this comment is relevant here, as it speaks about project drafts, etc
maybe needs to be updated